MainModule/Server/Core/Functions.lua

```diff
@@ -776,14 +776,14 @@ return function(Vargs, GetEnv)
 				end
 				local c = 0
 				for i = 1, 8 do
-					c = c + (sub(x, i, i) == '1' and 2 ^ (8 - i) or 0)
+					c += (sub(x, i, i) == '1' and 2 ^ (8 - i) or 0)
 				end
 				return char(c)
 			end))
 		end;
 
 		Hint = function(message,players,time)
-			for i,v in pairs(players) do
+			for _,v in ipairs(players) do
```

Comment:
Isn't `ipairs()` less performance than `pairs()`? as `ipairs()` skips nil values?

service.GetPlayers will never return nil values in it's table. Using `ipairs()` has no benefit.